Assets/Player/CameraControls.gd

```diff
@@ -0,0 +1,114 @@
+extends Node
+
+const ZOOM_IN_LIMIT = 5
+const ZOOM_OUT_LIMIT = 35
+const ZOOM_VALUE = 5
+const RAY_LENGTH = 1000
+
+const MOVE_SPEED = 10
+const MOVE_FASTER_MULT = 3
+
+export(NodePath) var origin_path : NodePath
+export(NodePath) var camera_path : NodePath
+export(NodePath) var rotation_y_path : NodePath
+
+onready var _origin : Spatial = get_node(origin_path)
+onready var _camera : Camera = get_node(camera_path)
+onready var _rotation_y : Spatial = get_node(rotation_y_path)
```

Comment:
When explicitly stating the type, do not include a space before the colon. (but do include a preceding space for `:=`)